asteroid/masknn/convolutional.py

```diff
@@ -400,12 +399,12 @@ class DCUNetComplexEncoderBlock(nn.Module):
         stride (Tuple[int, int]): Convolution stride.
         padding (Tuple[int, int]): Convolution padding.
         norm_type (str, optional): Type of normalization to use.
-            See ``asteroid.masknn.norms`` for valid values.
+            See :func:`~asteroid.masknn.norms` for valid values.
```

Comment:
```suggestion
            See :mod:`~asteroid.masknn.norms` for valid values.
```

It's a module right?

Reply:
Sorry, I committed that. And thanks for the suggestion.